static/src/inline-svgs/icon/hand.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
it should be.. https://github.com/guardian/frontend/blob/master/tools/__tasks__/compile/inline-svgs/index.js#L29